R/quasi-label.R

```diff
@@ -66,6 +65,39 @@ quasi_capture <- function(.quo, .label, .capture, ...) {
   act
 }
 
+auto_label <- function(expr, value) {
+  if (is.call(expr) || is.name(expr)) {
+    label <- expr_label(expr)
+    if (can_inline(value)) {
+      paste0(label, " (", as_label(value), ")")
+    } else {
+      label
+    }
+  } else {
+    expr_label(expr)
+  }
+}
+
+can_inline <- function(x) {
```

Comment:
`rlang::is_syntactic_literal()` might also be useful here? Would still need the nchar check.